Probe/test/test_data_collector.py

```diff
@@ -0,0 +1,28 @@
+import sys
```

Comment:
This file is redundant but keep it for the moment. You will write unit tests later. Anyway, create an entry point as `run.py` in the parent directory